server/Dockerfile

```diff
@@ -5,4 +5,4 @@ RUN npm install
 COPY . .
 EXPOSE 4000
 EXPOSE 4001
-CMD [ "npm", "start"]
+CMD [ "npm", "run", "server"]
```

Comment:
You are right.  Somehow I didn't run the build again, so the container was taking the old docker build which was running fine.😬